src/main/java/mate/service/impl/CarServiceImpl.java

```diff
@@ -0,0 +1,83 @@
+package mate.service.impl;
+
+import java.util.List;
+import mate.dao.CarDao;
+import mate.lib.Inject;
+import mate.lib.Service;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        List<Car> cars;
+        cars = carDao.getAll();
+        for (Car carItem : cars) {
+            if (car.getModel().equals(carItem.getModel())) {
+                return carItem;
+            }
+        }
+        return carDao.create(car);
```

Comment:
```suggestion
        return carDao.create(car);
```

А мы не можем создать 2 машины с одинаковыми моделями? Нам не нужно тут проводить эту проверку